src/appMain/start_external_proprietary.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+python sample_policy_manager.py --pack_port 8088 --unpack_port 8089 &
+POLICY_MANAGER=$!
```

Comment:
Not certain how easy it is to do, but it would be useful for the startup of SDL Core to fail if you don't have the necessary python packages installed (or at least give a warning with what went wrong). The README should also be updated with the python dependencies.